tests/navierStokesTempTurb/dynamicBoundaries/Test_NavierStokesTempTurbTunnel.xml

```diff
@@ -46,7 +46,7 @@
         <turbulence include = "Yes">
           <Pr_T> 0.5 </Pr_T>
         </turbulence>
-        <source type = "ExplicitEuler" temp_fct = "GaussST" ramp_fct = "RampTanh" dissipation = "No">
+        <source type = "ExplicitEuler" temp_fct = "Gauss" ramp_fct = "RampTanh" dissipation = "No">
```

Comment:
additional spaces

Reply:
db0eb82

Reply:
xml files are not up to date, caused by not being checked by test_xml-builder.